clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -51,21 +51,40 @@ Improvements to clangd
 Inlay hints
 ^^^^^^^^^^^
 
+- Type hints
+    * Improved heuristics for showing sugared vs. desguared types
+    * Some hints which provide no information (e.g. ``<dependent-type>``) are now omitted
+- Parameter hints
+    * Parameter hints are now shown for calls through function pointers
+    * Parameter hints are now shown for calls to a class's ``operator()``
+    * No longer show bogus parameter hints for some builtins like ``__builtin_dump_struct``
+
 Diagnostics
 ^^^^^^^^^^^
 
-Semantic Highlighting
-^^^^^^^^^^^^^^^^^^^^^
+- Improved quality of include-cleaner diagnostics (missing and unused includes)
```

Comment:
i'd move this one into include-cleaner release notes (I am also happy to do that myself), a more detailed version could look like:

- Support for `--only-headers` flag to limit analysis to headers matching a regex
- Recognizes references through `concept`s
- Builtin headers are not analyzed
- Handling of references through `friend` declarations
- Fixes around handling of IWYU pragmas on stdlib headers
- Improved handling around references to/from template specializations

Reply:
Thanks, I added these in an include-cleaner section